### PR TITLE
Manually contribute featured.json file

### PR DESCRIPTION
**Pull request by ericwill**
Signed-off-by: Eric Williams <ericwill@redhat.com>

### What does this PR do?
Manually contributes a featured.json file, needed for plugin recommendations to work.

### What issues does this PR fix or reference?
CRW-1672

### CODE REVIEW

File: dependencies/che-plugin-registry/v3/plugins/eclipse/che-theia/featured.json [hunk 1]
```diff
@@ -0,0 +1,227 @@
+{
```
[the remark this answers is not in the document]
Comment (ericwill) [reply dependencies/che-plugin-registry/v3/plugins/eclipse/che-theia/featured.json:1]: oops, will fix it

File: dependencies/che-plugin-registry/v3/che-theia/featured.json [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+          "languages": [
+            {
+              "id": "xml"
+            }
+          ]
+        }
+      }
+    ]
+  }
+
```
Comment (benoitf) [inline dependencies/che-plugin-registry/v3/che-theia/featured.json:227]: missing newline

**Review by benoitf — APPROVED** [review]
LGTM

**Comment by chenshibi** [comment]
试试


| |
陈思必
|
|
***@***.***
|
签名由网易邮箱大师定制
On 3/19/2021 04:36，Eric ***@***.***> wrote：

Merged #452 into crw-2-rhel-8.

**Pull request merged**